webapp/src/services/api/constants.ts

```diff
@@ -2,3 +2,4 @@ const API_URL_BASE = "v1";
 const STUDIES_API_URL = `${API_URL_BASE}/studies/{studyId}`;
 
 export const TABLE_MODE_API_URL = `${STUDIES_API_URL}/tablemode`;
+export const RAW_API_URL = `${STUDIES_API_URL}/raw`;
```

Comment:
in my opinion you can remove it it's not worth it to over optimize just to replace: `v1/studies/${studyId}/raw/download`

Reply:
The goal is to used it for the other Raw API (I don't make it in this PR) instead of repeat it.